src/components/RegionSelect/RegionSelect.tsx

```diff
@@ -77,6 +139,23 @@ export default function RegionSelect() {
           },
         }}
         renderInput={(params) => <TextField {...params} />}
+        renderOption={(props, option) => {
+          const { key, ...otherProps } = props
+          return (
+            <li
+              key={key}
+              {...otherProps}
+              style={{
+                opacity:
+                  option.label === noCountriesMatchText || option.label === noRegionsMatchText
+                    ? 0.6
+                    : 1,
```

Comment:
Non-blocking suggestion:
```suggestion
                  opacity: option.label === (noCountriesMatchText || noRegionsMatchText) ? 0.6 : 1,
```

Reply:
this is invalid, the suggestion only checks the noCountriesMatchText. 
Mine is checking both.